Gitweb:     http://git.kernel.org/linus/41855b77547fa18d90ed6a5d322983d3fdab1959
Commit:     41855b77547fa18d90ed6a5d322983d3fdab1959
Parent:     2ae8bb75db1f3de422eb5898f2a063c46c36dba8
Author:     Joe Perches <joe@perches.com>
AuthorDate: Mon Nov 9 17:58:50 2009 -0800
Committer:  Ingo Molnar <mingo@elte.hu>
CommitDate: Tue Nov 10 06:05:39 2009 +0100

    x86: GART: pci-gart_64.c: Use correct length in strncmp
    
    Signed-off-by: Joe Perches <joe@perches.com>
    Cc: <stable@kernel.org> # .3x.x
    LKML-Reference: <1257818330.12852.72.camel@Joe-Laptop.home>
    Signed-off-by: Ingo Molnar <mingo@elte.hu>
---
 arch/x86/kernel/pci-gart_64.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/x86/kernel/pci-gart_64.c b/arch/x86/kernel/pci-gart_64.c
index a9bcdf7..eb46ab3 100644
--- a/arch/x86/kernel/pci-gart_64.c
+++ b/arch/x86/kernel/pci-gart_64.c
@@ -858,7 +858,7 @@ void __init gart_parse_options(char *p)
 #endif
 	if (isdigit(*p) && get_option(&p, &arg))
 		iommu_size = arg;
-	if (!strncmp(p, "fullflush", 8))
+	if (!strncmp(p, "fullflush", 9))
 		iommu_fullflush = 1;
 	if (!strncmp(p, "nofullflush", 11))
 		iommu_fullflush = 0;
